### ISSUE TITLE
update due to new convention on array indexing in Julia 0.5.0

### ISSUE THREAD

**Issue by konkam** [dennisprangle/ABCDistances.jl#1]
Hello,

first I'd like to thank you for this package which I have found useful.

I've noticed a first incompatibility issue with Julia 0.5.0 due to the new convention on array indexing.

The information comes from here [https://github.com/JuliaLang/julia/blob/master/NEWS.md](url), 

the relevant part is pasted here:

> Breaking changes
> 
> All dimensions indexed by scalars are now dropped, whereas previously only trailing scalar dimensions would be omitted from the result (#13612). This is a very major behavioral change, but should cause obvious failures. To retain a dimension sliced with a scalar i slice with i:i instead.

The error stems from line 150 of abc.jl:

```
for i in 1:p
    y = squeeze(out.parameters[i,:], 1)
    (CI_lower[i], CI_upper[i]) = quantile(y, WeightVec(out.weights), [0.025,0.975])
end
```

a quick fix is:

```
for i in 1:p
    (CI_lower[i], CI_upper[i]) = quantile(out.parameters[i,:], WeightVec(out.weights), [0.025,0.975])
end
```

Best,

edit:

another fix needed is to replace in abc_pmc.jl, line 552:

`currvar = cov(x.parameters, wv, vardim = 2)`

by:

`currvar = cov(x.parameters, wv, 2)`

**Comment by dennisprangle** [comment]
Brilliant, thanks for this! I hadn't got round to installing 5.0 yet. I'll make these fixes over the next couple of days.

**Comment by konkam** [comment]
Glad I could help, and thanks again for the good work on this package

**Comment by dennisprangle** [comment]
It took a while to test all my scripts, but this is (hopefully) fixed now in v0.1.1.

**Comment by konkam** [comment]
Hello,

I tried to run the new version of the package and everything seems also good on my side.

thanks
